src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -69,8 +69,19 @@ internal async Task ProcessHttpInAndOutInvocationRequest(ScriptInvocationContext
 
             try
             {
-                using (HttpRequestMessage httpRequestMessage = CreateAndGetHttpRequestMessage(scriptInvocationContext.FunctionMetadata.Name, scriptInvocationContext.ExecutionContext.InvocationId.ToString(), new HttpMethod(httpRequest.Method), httpRequest.GetQueryCollectionAsDictionary()))
+                using (HttpRequestMessage httpRequestMessage = CreateAndGetHttpRequestMessage(scriptInvocationContext.FunctionMetadata.Name, scriptInvocationContext.ExecutionContext.InvocationId.ToString(), new HttpMethod(httpRequest.Method), httpRequest.GetQueryCollectionAsDictionary(), httpRequest.GetRequestUri()))
                 {
+                    httpRequestMessage.Content = new StreamContent(httpRequest.Body);
+
+                    if (!string.IsNullOrEmpty(httpRequest.ContentType))
+                    {
+                        httpRequestMessage.Content.Headers.Add("Content-Type", httpRequest.ContentType);
+                    }
+                    if (httpRequest.ContentLength != null)
+                    {
+                        httpRequestMessage.Content.Headers.Add("Content-Length", httpRequest.ContentLength.ToString());
+                    }
```

Comment:
Does this mean other headers will be stripped out? I think all headers should be forwarded.

Reply:
Ideally we need to avoid rebuilding each part in http request when forwarding. Using `HttpRequestMessageFeature` gave us that ability. Http request will be forwarded and only additional headers from host will be added.

Reply:
All headers are forwarded now.